content/code_samples/rippleapi_quickstart/get-account-info.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const {RippleAPI} = require('ripple-lib');
+
+const api = new RippleAPI({
+  server: 'wss://s1.ripple.com' // Public rippled server
+});
+api.connect().then(() => {
+  /* begin custom code ------------------------------------ */
+  const my_address = 'rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn';
```

Comment:
RippleAPI and javascript both use camelCase, I think it would be a bit more aesthetic to use it in the examples too.